kolibri/core/assets/src/views/app-body.vue

```diff
@@ -2,7 +2,12 @@
 
   <!-- class unused, used as identifier when debugging from DOM -->
   <div class="app-body" :style="contentStyle">
-    <loading-spinner v-if="loading" />
+    <k-indeterminate-linear-indicator
+      v-if="loading"
+      class="toolbar-loading-indicator"
+      :style="{ top: padding === 32 ? '64px' : '56px' }"
```

Comment:
This seems pretty brittle to me.

Would recommend switching the prop on `app-body` to be a boolean `isMobile`, and then defining the padding and other styling specifics internally rather than passing in `padding` from the parent.

Alternatively, remove the prop completely and just use `responsiveWindow` in here. That'd probably be cleanest.